ocpp/messages.py

```diff
@@ -182,6 +182,53 @@ def get_validator(
     # character. If no encoding is given, reading the schema would fail with:
     #
     #     Unexpected UTF-8 BOM (decode using utf-8-sig):
+    try:
+        with open(path, 'r', encoding='utf-8-sig') as f:
+            data = f.read()
+            validator = Draft4Validator(json.loads(data,
+                                                   parse_float=parse_float))
+            _validators[cache_key] = validator
+        return _validators[cache_key]
+    except FileNotFoundError:
+        return _get_backport_validator(
+            message_type_id,
+            action,
+            ocpp_version,
+            parse_float
+        )
+
+
+def _get_backport_validator(
+        message_type_id: int,
+        action: str,
+        ocpp_version: str,
+        parse_float: Callable = float
+) -> Draft4Validator:
+    """
+    Reads backported schema from disk.
+
+    OCPP 1.6 Improved Security specification has selected Requests and
+    Responses backported from 2.0 to 1.6.
+    """
+    if ocpp_version not in ["1.6"]:
+        raise ValueError
```

Comment:
Can you add context to the ValueError. Otherwise it will be harder to understand what was the error exactly

Reply:
This was my mistake. I thought I had removed the _get_backport_validator but apparently did some mistake and it was still there.